app/models/concerns/decision_form_persistance.rb

```diff
@@ -2,62 +2,12 @@ module DecisionFormPersistance
   extend ActiveSupport::Concern
 
   included do
-    include ActiveModel::Model
-    include ActiveModel::Attributes
-    include ActiveRecord::ReadonlyAttributes
-    include ActiveRecord::AttributeAssignment
+    include FormPersistance
```

Comment:
Extracted form persistence from decision specific component so that it can be used elsewhere.